conjure-java-core/src/main/java/com/palantir/conjure/java/types/EnumGenerator.java

```diff
@@ -54,33 +63,44 @@ public final class EnumGenerator {
     private static final String VISIT_METHOD_NAME = "visit";
     private static final String VISIT_UNKNOWN_METHOD_NAME = "visitUnknown";
     private static final TypeVariableName TYPE_VARIABLE = TypeVariableName.get("T");
+    public static final String UNKNOWN_TYPE_PARAM_NAME = "unknownType";
+    static final String UNKNOWN_TYPE_NAME = "unknown";
```

Comment:
Good catch, `private static final` would be better, I don't think there are uses in other classes. Similarly the `UNKNOWN_TYPE_PARAM_NAME` above should also be private.

Reply:
(it's fine if this releases as is I'd be happy to approve a fixup later since this module isn't used as a library, so we don't have to worry about folks consuming internal fields)